rate_control.md

```diff
@@ -1,5 +1,31 @@
 ---
-title: Rate Control
+title: Rate Control (WIP)
 ---
 
-# Rate Control
+# Rate Control (WIP)
+
+## Overview
+
+The rate control protocol is the key part of the ultimate goal of UCSMA, which attempts to maintain fairness among different nodes in an ad-hoc wireless network. There are two parts of the rate control protocol:
```

Comment:
Maybe the goal of UCSMA should be its own paragraph at the top?

Reply:
Maybe add that later, but it's a needed context here